lib/hanami/slice_registrar.rb

```diff
@@ -97,16 +104,13 @@ def parent_slice_namespace
       parent.eql?(parent.app) ? Object : parent.namespace
     end
 
-    # Runs when a slice file has been found at `config/slices/[slice_name].rb`, or a slice directory
-    # at `slices/[slice_name]`. Attempts to require the slice class, if defined, before registering
-    # the slice. If a slice class is not found, registering the slice will generate the slice class.
+    # Runs when a slice file has been found at `config/slices/[slice_name].rb`, or else inside
+    # the slice at `slices/[slice_name]/config/slice.rb`, or a slice directory at `slices/[slice_name]`.
```

Comment:
Technically, "or else inside the slice at `slices/[slice_name]/config/slice.rb`" doesn't really apply here, since we only get to the point of running `load_slice` if only one of these two conditions are met?

1. slice file has been found at `config/slices/[slice_name].rb`, OR
2. a slice directory has been found at `slices/[slice_name]`

The job of looking for the slice file inside `slices/[slice_name]/config/slice.rb` happens _within_ this method rather than _before_ it, right? So perhaps we could tighten up the comment here to make sure it's not misleading at all?